src/transformers/trainer_pt_utils.py

```diff
@@ -1170,7 +1170,9 @@ class AcceleratorConfig:
             training results are fully reproducable using a different sampling technique. While seed-to-seed results
             may differ, on average the differences are neglible when using multiple different seeds to compare. Should
             also be ran with [`~utils.set_seed`] for the best results.
-
+        gradient_accumulation_kwargs (`dict`, *optional*, defaults to `None`):
```

Comment:
Let's fully document these parameters please/acceptable keys.

Reply:
Sure are you thinking of something like this?

```
"""
gradient_accumulation_kwargs (`dict`, *optional*):
    Additional kwargs to configure gradient accumulation, see [`accelerate.utils.GradientAccumulationPlugin`]. Any 
    of the following (optional) keys are acceptable:
        num_steps (`int`): Will take precedence over [`~.TrainingArguments.gradient_accumulation_steps`] if 
        the latter is set to 1, otherwise an exception will be raised.
        adjust_scheduler (`bool`): Whether to adjust the scheduler steps to account for [`~.TrainingArguments.gradient_accumulation_steps`].
        The [`accelerate.utils.GradientAccumulationPlugin`] default is `True`.
        sync_each_batch (`bool`): Whether to synchronize the gradients at each data batch. 
        The [`accelerate.utils.GradientAccumulationPlugin`] default is `False`.

    The following key has no effect and will be treated as if it is not passed.
        sync_with_dataloader (`bool`): Will be ignored and always set to `False`.
"""
```

Reply:
Yes, bar some tabs which the formatter should get you on.

(Also no need to have `sync_with_dataloader`)